src/olympia/activity/views.py

```diff
@@ -62,7 +62,7 @@ def get_serializer_context(self):
     @waffle_switch('activity-email')
     def create(self, request, *args, **kwargs):
         version = self.get_version_object()
-        if not version == version.addon.latest_or_rejected_version:
+        if version != version.addon.find_latest_version_including_rejected():
```

Comment:
Note to self: change this one too, pass the channel from `version` in the call. Need to fix `find_latest_version_including_rejected()` to accept a `channel` argument first, obviously...